src/components/form/form_row/make_id.ts

```diff
@@ -1,6 +1,9 @@
 // Generate statistically almost-certainly-unique `id`s for associating form
 // inputs with their labels and other descriptive text elements.
 function makeId(): string {
+  console.log(
+    'WARNING: makeId is deprecated. Use htmlIdGenerator from services instead.'
```

Comment:
`htmlIdGenerator` is exported from the top level, the message can direct the developer a little better

```suggestion
    'WARNING: makeId is deprecated. Use htmlIdGenerator from @elastic/eui instead.'
```